install_config/configuring_vsphere.adoc

```diff
@@ -157,6 +157,27 @@ determined by using the command `uname -n`. In case of a mismatch, the native
 cloud integration for those providers will not work.
 ====
 
+== Configuring OpenShift Container Platform for vSphere using Ansible
```

Comment:
Unless "OpenShift Container Platform for vSphere" is a proper name for a component, s/Configuring OpenShift Container Platform/{product-title} (all places in this addition)

Reply:
you mean `s/OpenShift Container Platform/{product-title}` right ? I'm also done !